pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -63,6 +64,7 @@ const (
 	ElasticsearchKubeControllersUserName               = "tigera-ee-kube-controllers"
 	ElasticsearchKubeControllersSecureUserSecret       = "tigera-ee-kube-controllers-elasticsearch-access-gateway"
 	ElasticsearchKubeControllersVerificationUserSecret = "tigera-ee-kube-controllers-gateway-verification-credentials"
+	KubeControllerPrometheusTLSSecret                  = "tigera-kube-controller-prometheus-tls"
```

Comment:
I'd call it `calico-kube-controllers-metrics-tls`, I suspect sooner or later this might be supported for both Variants. Either way, the calico-kube-controllers deployment will also support it for Enterprise as well.

Reply:
Done